siatune/core/hooks/reporter.py

```diff
@@ -1,18 +1,29 @@
 # Copyright (c) SI-Analytics. All rights reserved.
+import glob
+import os
+from os import path as osp
 from pathlib import Path
 from typing import Dict, Optional, Union
 
-from mmengine.dist import get_dist_info
-from mmengine.hooks import LoggerHook
-from mmengine.registry import HOOKS
-from mmengine.runner import Runner
 from ray.air import session
-from torch import distributed as dist
+from ray.air.checkpoint import Checkpoint
 
 from siatune.version import IS_DEPRECATED_MMCV
 
+
+def get_latest_ckpt(work_dir: str) -> dict:
+    files = glob.glob(osp.join(work_dir, '*.pth'))
+    if not files:
+        return dict()
+    return dict(path=max(files, key=os.path.getctime))
+
+
 if not IS_DEPRECATED_MMCV:
+    from mmengine.dist import master_only
+    from mmengine.hooks import LoggerHook
     from mmengine.hooks.logger_hook import SUFFIX_TYPE
+    from mmengine.registry import HOOKS
+    from mmengine.runner import Runner
 
     @HOOKS.register_module()
     class RayTuneLoggerHook(LoggerHook):
```

Comment:
How about renaming it to `RayTuneReporterHook`?
Current name may confuse users to understand it as `LoggerHook` only.